.evergreen/preinstall.sh

```diff
@@ -12,6 +12,11 @@ echo "NPM_VERSION: $NPM_VERSION"
 echo "APPDATA: $APPDATA"
 echo "PATH: $PATH"
 
+# these are super useful if you want to run the smoke tests locally
```

Comment:
We're going to need these when we want to test with the files produced by a PR or beta or ga build, so I'm just adding it next to other useful/essential variables being output. They aren't secrets so that's fine.